modules/nw-openstack-hw-offload-testpmd-pod.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: REFERENCE
+[id="nw-openstack-hw-offload-testpmd-pod_{context}"]
```

Comment:
@EmilienM Would you also like to include an HW offload testpmd reference with this update?

Reply:
yes, please. Do you need me to provide one for you?

Reply:
I pulled [this](https://github.com/openshift/openshift-docs/pull/46276/files/5a8c8afe840728a1b3dc87e1447c4be667f5faaf#diff-510b7fd40cc6117545c617a4d55532bdc49790d8d3ce3ca5537155231a4ef080) from upstream. Is that fine by you?

Reply:
you need to add the `runtimeClassName` thing for PerformanceProfile and you're good